_includes/author_blurb.html

```diff
@@ -0,0 +1,13 @@
+{% if page.author %}
+	{% if page.author_avatar %}
+		<img src="/assets/{{ page.author_avatar }}" alt="" class="br-100 w3 dib">
```

Comment:
https://www.youtube.com/watch?v=SsoOG6ZeyUI 😂 

Which editor do you use? Perhaps we could have an .editorconfig — more info here http://EditorConfig.org

Reply:
I use PHP storm and sometimes Atom. The tabs should be converted to 4 spaces automatically. Not sure what the standard is for liquid templates. I don't think that there's one for liquid templates and markdown. In PHP, the standards are 4 spaces so I configure the IDE to convert the Tab character as 4 spaces.

Reply:
I don't think I'll be doing templating work that often on K8s and markdown posts don't require any tabs or spaces so it's up to you if you want the hassle of a config. If so just add it to the repo and I'll fetch 🐶

Reply:
👍 for the video from silicon valley. I think we both agree that pressing the space bar multiple times is silly! However, in PHP PSR-2 standard, we use 4 spaces. Thus I set my IDE to automatically put 4 spaces chars when I use the Tab char. In HTML, I use 2 spaces per indent and I also have it setup to change the Tab char to the 2 spaces.

Tab char can very in cols, spaces don't. I think that the gist of it from the PSR-2 perspective https://www.php-fig.org/psr/psr-2/

Reply:
I use 2 spaces :) too. I think the partial has 4?

Reply:
I checked... one indent = 2 spaces, 2 indents = 4. Github is displaying it this way. The file is using 2 spaces per indent.

![image](https://user-images.githubusercontent.com/2071560/35797468-baf42b7a-0a57-11e8-939f-b0086759d711.png)

Reply:
Tab is not expanded to 2 spaces: https://www.dropbox.com/s/voiapr0ay618osn/Kapture%202018-02-05%20at%2010.06.45.mp4?dl=0

Reply:
I already tried this several times. If I merge, git does not notice the differences. I'll just edit it here so we can merge this.